x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/math/PowTests.java

```diff
@@ -185,8 +185,8 @@ public static Iterable<Object[]> parameters() {
                 )
             ),
             new TestCaseSupplier("pow(integer, double)", () -> {
-                // Negative numbers to a non-integer power are NaN
-                int base = randomIntBetween(0, 1000);
+                // Positive numbers to a non-integer power
+                int base = randomIntBetween(1, 1000);
```

Comment:
@not-napoleon @craigtaverner I could have interpreted the expectation of this scenario incorrectly. If so, then what is the intent, as the previous comment and code seem misaligned.